filters/include/pcl/filters/impl/random_sample.hpp

```diff
@@ -57,7 +57,7 @@ pcl::RandomSample<PointT>::applyFilter (PointCloud &output)
     copyPointCloud (*input_, output);
     // Get X, Y, Z fields
     std::vector<pcl::PCLPointField> fields;
-    pcl::getFields (*input_, fields);
+    pcl::getFields<PointT> (fields);
```

Comment:
Now that we have a single output parameter, this looks awkward. What about adding an overload that returns a vector?

Reply:
Future PR?